lib/server/env/index.ts

```diff
@@ -9,9 +12,16 @@ export function getEnv(): Env {
     throw new Error('Missing GATEWAY JWT SECRET');
   }
 
+  if (!process.env.inxtbridge_storage__JWT_SECRET) {
+    throw new Error('Missing STORAGE JWT SECRET');
+  }
+
   return {
     gateway: {
       jwtSecret: process.env.inxtbridge_gateway__JWT_SECRET
+    },
+    storage: {
+      jwtSecret: process.env.inxtbridge_storage__JWT_SECRET
```

Comment:
Is this the same jwt secret that has to be in the farmer?

Reply:
yep